test/ui/test_spec.js

```diff
@@ -96,29 +94,42 @@ describe('UI tests', function() {
     });
 
     it('appears correctly before any actions have been taken', function(done) {
-      return screenshotAndCompare('body.png', 'body', done);
+      screenshotAndCompare('body.png', 'body')
```

Comment:
I find it interesting that in test_spec.js you move 'done' down into the function being called, whereas here you are doing just the opposite.